ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.12.yaml

```diff
@@ -6,7 +6,6 @@ base_images:
 binary_build_commands: make build
 build_root:
   from_repository: true
-  use_build_cache: true
```

Comment:
@petr-muller please comment on that change.

Reply:
Comment in what sense? I need at least some context

Reply:
sorry, please see the last message in the thread.

Reply:
https://github.com/openshift/release/pull/26414#issuecomment-1058008871